verify-service/src/main/java/io/mosip/verifyservice/services/VerifiablePresentationRequestServiceImpl.java

```diff
@@ -0,0 +1,87 @@
+package io.mosip.verifyservice.services;
+
+import io.mosip.verifycore.dto.authorizationRequest.AuthorizationRequestCreateDto;
+import io.mosip.verifycore.dto.authorizationRequest.AuthorizationRequestCreateResponseDto;
+import io.mosip.verifycore.dto.authorizationRequest.AuthorizationRequestDto;
+import io.mosip.verifycore.dto.presentation.PresentationDefinitionDto;
+import io.mosip.verifycore.enums.SubmissionState;
+import io.mosip.verifycore.models.AuthorizationRequestCreateResponse;
+import io.mosip.verifycore.models.PresentationDefinition;
+import io.mosip.verifycore.shared.Constants;
+import io.mosip.verifycore.spi.VerifiablePresentationRequestService;
+import io.mosip.verifycore.utils.SecurityUtils;
+import io.mosip.verifycore.utils.Utils;
+import io.mosip.verifyservice.repository.AuthorizationRequestCreateResponseRepository;
+import io.mosip.verifyservice.repository.PresentationDefinitionRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import java.time.Instant;
+import java.util.Timer;
+import java.util.TimerTask;
+
+import static io.mosip.verifycore.shared.Config.DEFAULT_EXPIRY;
+
+@Service
+public class VerifiablePresentationRequestServiceImpl implements VerifiablePresentationRequestService {
+
+    @Autowired
+    PresentationDefinitionRepository presentationDefinitionRepository;
+    @Autowired
+    AuthorizationRequestCreateResponseRepository authorizationRequestCreateResponseRepository;
+    public VerifiablePresentationRequestServiceImpl() {}
+
+    @Override
+    public AuthorizationRequestCreateResponseDto createAuthorizationRequest(AuthorizationRequestCreateDto vpRequestCreate) {
+
+        String transactionId = vpRequestCreate.getTransactionId()!=null ? vpRequestCreate.getTransactionId() : Utils.createID(Constants.TRANSACTION_ID_PREFIX);
+        String requestId = Utils.createID(Constants.REQUEST_ID_PREFIX);
```

Comment:
For the same transaction, if vp-request endpoint is called multiple times, we don't restricts ? How will you know which request id submission should be considered ?

Reply:
submission is based on req-id which is unique and there can be only one submission at a time

Reply:
the questions is what happens when the create authorization request is called multiple times ? 
Will it create a new request id, when the old request id is still valid ?

Reply:
Each time it will create a new req_id. 

All the req_id is still valid because it represents a new transaction every time. 

Its the verify FE tries to submit for latest req_id.

Reply:
??